jerry-core/include/jerryscript-port.h

```diff
@@ -150,6 +150,30 @@ double jerry_port_get_current_time (void);
  */
 struct jerry_instance_t *jerry_port_get_current_instance (void);
 
+/*
+ * Debugger Port API
+ */
+
+/* forward declaration */
+struct jerry_debugger_uint8_data_t;
```

Comment:
This forward declaration also seems to be unnecessary, especially in this public header. The type is only used in sources internal to jerry-core. If I see it right.